src/test/java/uk/gov/hmcts/darts/common/service/impl/DataAnonymisationServiceImplTest.java

```diff
@@ -87,16 +87,58 @@ private void assertLastModifiedByAndAt(CreatedModifiedBaseEntity entity, UserAcc
 
 
     @Test
-    void positiveEventEntityAnonymise() {
+    void positiveEventEntityAnonymiseNotUpdatedAsNotAllCasesExpiredAndOnlyAnonymiseIfAllCasesExpiredIsTrue() {
```

Comment:
Why do you need the word 'positive' in front of every test? What purpose is that serving?

Reply:
Its a test term:

positive = Indicates the test is verifying things we expect the application to do
negative = Indicates the test is verifying things we do not expect the application to do (error scenarios and the such)

Reply:
Ok I still don't get the benefit of this and seems odd that this has been introduced as a pattern when it hasn't been done by anyone else in the code base. 
I keep re-reading these test names and really struggling to get my head round what you are trying to say. For someone new coming in years down the line they're not going to have a clue. Please can you try and use some of the patterns we already have in place for test names?

Reply:
Worth a chat in best-practices I think to refine some common naming pattern. I know we tried this a while back and didn't get very far, but now that we're fewer in numbers it's worth revisiting. I'll initiate something.

Reply:
Its probs also worth using nested classes as it helps structure the test cases